src/main/kotlin/espressoLogcat/EspressoLogcat.kt

```diff
@@ -6,254 +6,245 @@ import java.io.InputStreamReader
 import java.time.format.DateTimeFormatter
 
 class EspressoLogcat {
+    /**
+     * Settings Class is used to set static variables that are used when getting the logcat
+     */
+    class Settings {
+        companion object {
```

Comment:
Are you abandoning this PR and just doing the other one instead where you move from static -> Instance variables?

Reply:
Ah yeah. I think the original plan was to have this one up as it is smaller and could be reviewed quicker, but after working through some of the Interface problems, we decided it made more sense to have them as Instance variables. I'll abandon this.